RecoLocalCalo/HcalRecAlgos/test/HcalSevLvlAnalyzer.cc

```diff
@@ -88,14 +94,11 @@ void HcalSevLvlAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup
 #endif
 
 #ifdef THIS_IS_AN_EVENTSETUP_EXAMPLE
-  ESHandle<SetupData> pSetup;
-  iSetup.get<SetupRecord>().get(pSetup);
+  auto pSetup = &iSetup.getData(tokSet_);
 #endif
```

Comment:
Now that the event setup happens with the `getData` method, I believe these lines are not needed anymore

Reply:
removed